app/src/main/java/pl/szczodrzynski/edziennik/App.kt

```diff
@@ -172,6 +175,7 @@ class App : MultiDexApplication(), Configuration.Provider, CoroutineScope {
         App.profile = Profile(0, 0, 0, "")
         debugMode = BuildConfig.DEBUG
         devMode = config.debugMode || debugMode
+        chucker = config.chucker
```

Comment:
Enable by default when `devMode` is enabled.

Reply:
done